config.go

```diff
@@ -52,15 +54,14 @@ type AwsConfig struct {
 }
 
 func loadConfig() Config {
-	godotenv.Load(".env")
-	if os.Getenv("DB_ENGINE") == "" {
-		os.Setenv("DB_ENGINE", "lmdb")
-	}
+	_ = godotenv.Load(".env")
 
 	return Config{
 		OwnerNpub:                        getEnv("OWNER_NPUB"),
-		DBEngine:                         getEnv("DB_ENGINE"),
+		DBEngine:                         getEnvString("DB_ENGINE", "lmdb"),
```

Comment:
Why was this changed? It should default to badger

Reply:
It was already defaulting to LMDB when no env parameter was set. Look at line 57 above. I assumed you did this intentionally - wanting new installs to default to Badger, but not to touch old installs already using LMDB without specifying the env variable. If you'd prefer to switch even old installs to use Bager by default, feel free to change the string, or let me know and I can make the change myself.

Reply:
Ah yes you're right lol, thanks for knowing my relay better than me. Too many relays! Thanks for the pr